docs/_data/2017.eu.speakers.yaml

```diff
@@ -509,7 +509,7 @@
     title: 'Tech Writers Without Borders: Making the world a better place, one (numbered) step at a time'
     slides: https://speakerdeck.com/sculshaw/making-the-world-a-better-place-one-numbered-step-at-a-time
     event: Write the Docs EU 2017
-    path: conf/eu/2017/videos/
+    path: conf/eu/2017/videos/tech-writers-without-borders-making-the-world-a-better-place-one-numbered-step-at-a-time-stuart-culshaw
     series: Write the Docs EU
     series_slug: eu
     slug: ''
```

Comment:
@bethaitman  I reckon this is the problem, not the path. Or as well as the path.

Reply:
Genius. I didn't spot that